src/Polly.Strategies/Predicates.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Polly;
+
+public delegate ValueTask<bool> PredicatesDelegate<T, TContext>(Outcome<T> outcome, TContext context);
+
+public class Predicates<TContext>
+{
+    internal Dictionary<Type, List<object>> Callbacks { get; } = new();
+
+    public Predicates<TContext> AddException<T>()
```

Comment:
Can all these single-line functions be coded with expression syntax for brevity?